src/main/java/uk/gov/hmcts/reform/professionalapi/controller/external/ProfessionalExternalUserController.java

```diff
@@ -55,26 +56,29 @@
     @ApiParam(
             name = "showDeleted",
             type = "string",
-            value = "flag (True/False) to decide deleted users needs to be shown",
-            required = false
+            value = "Flag (True/False) to decide whether Deleted Users are included in the response"
     )
     @ApiResponses({
             @ApiResponse(
                     code = 200,
-                    message = "List of a professional users along with details",
-                    response = ProfessionalUsersEntityResponse.class
+                    message = "List of Professional Users and their details",
+                    response = ProfessionalUsersResponse.class
```

Comment:
it should be ProfessionalUsersEntityResponse

Reply:
ProfessionalUsersEntityResponse does not provide an Example Model in Swagger, it only shows as:

{}

which is not useful, so using ProfessionalUsersResponse to provide a more useful Example Model

Reply:
can you check how list of contactInformation is visible in swagger sample response? Both are list under same object. 
This could be because In ProfessionalUsersEntityResponse list is initialised to empty. Please try removing initialisation and see if you can see list of users sample response.

Reply:
Will attempt this 👍

Reply:
removing initialisation resulted in failing Unit Tests with nullPointerException

attempting to see if @JsonProperty on the List will make it visible in Swagger

Reply:
use responseContainer = "list" and class as ProfessionalUsersResponse

Reply:
Thanks that worked and has been implemented in this commit: https://github.com/hmcts/rd-professional-api/pull/503/commits/112cdd4947432ed0c289a23196b9343ab71be578